exchanges/account/account.go

```diff
@@ -63,12 +65,14 @@ func SubscribeToExchangeAccount(exchange string) (dispatch.Pipe, error) {
 	return service.mux.Subscribe(accounts.ID)
 }
 
-// Process processes new account holdings updates
-func Process(h *Holdings) error {
-	return service.Update(h)
+// Process processes new account holdings updates and returns the resultant.
+func Process(h *Holdings, c *Credentials) error {
+	return service.Update(h, c)
 }
 
-// GetHoldings returns full holdings for an exchange
+// GetHoldings returns full holdings for an exchange.
+// NOTE: Due to credentials these amounts could be N*APIKEY actual holdings.
+// TODO: Add jurisdiction and differentiation between APIKEY holdings.
```

Comment:
Yeah, I'm guessing what this means is when I actually call
`gctcli --apikey="subaccount" --apisecret="subaccount"  getaccountinfo ftx spot`
I only receive data related to my subaccount? At present I get `main` and `subaccount`